main/guides/getting-started/README.md

```diff
@@ -9,7 +9,8 @@ After installing the Agoric SDK, you can proceed to [starting a project](./start
 If you're familar with JavaScript development tools such as `node`, `yarn`, and `git`:
 
 ```shell
-node --version # 16.19.1 or higher
+brew install jq # Install jQuery
```

Comment:
`brew` isn't available on linux.

Also, jQuery and jq are different things.

Reply:
Please let's move any changes to the getting-started section to its own PR.

Reply:
This has been moved to another PR. The installation instructions have also been made platorm-agnostic.